staging/src/k8s.io/apiserver/pkg/apis/config/v1/types.go

```diff
@@ -33,7 +33,8 @@ type EncryptionConfiguration struct {
 
 // ResourceConfiguration stores per resource configuration.
 type ResourceConfiguration struct {
-	// resources is a list of kubernetes resources which have to be encrypted.
+	// resources is a list of kubernetes resources which have to be encrypted. The resource names are derived from `resource` or `resource.group` of the gvr.
```

Comment:
Can we provide a complete example above `EncryptionConfiguration`?  I always have to look at the tests to remember the complete schema.